modules/reactor/test/tests/meshgenerators/azimuthal_block_split_generator/tests

```diff
@@ -64,6 +79,21 @@
       expect_err = 'This parameter contains blocks that do not exist in the input mesh.'
       detail = "if old_blocks contains blocks that do not exist in the input mesh."
     []
+    [error_mod_center_quad_elements]
+      type = 'RunException'
+      input = 'azi_block_id_mod.i'
+      cli_args = 'Mesh/cd/quad_center_elements="true"
+                  Mesh/cd/num_sectors_per_side="4 4 4 4 4 4"
+                  Mesh/cd/ring_block_ids="10 10 20"
+                  Mesh/cd/ring_block_names="center center cd_ring"
+                  Mesh/cd_azi_define/old_blocks="10 20"
+                  Mesh/cd_azi_define/new_block_ids="100 200"
+                  Mesh/cd_azi_define/new_block_names="center_new cd_ring_new"
+                  --mesh-only "azi_block_id_mod_err.e"'
+      expect_err = 'This parameter contains a block that involves center quad elements, the nodes of which should not be moved to match the given angles.'
+      detail = "if old_blocks contains center quad elements."
```

Comment:
```suggestion
      detail = "if the list of blocks whose nodes should be moved contains a block with center quad elements."
```

Reply:
we try to avoid naming parameters in the requirements. Parameters could change names and we could forget to update requirements. It s also less readable by someone who would be auditing only